libs/market-depth/src/lib/orderbook.tsx

```diff
@@ -99,12 +112,67 @@ export const Orderbook = ({
   const groupedBids = useMemo(() => {
     return compactRows(bids, VolumeType.bid, resolution);
   }, [bids, resolution]);
+  const [isOpen, setOpen] = useState(false);
+  const previousMidPrice = usePrevious(midPrice);
+  const iconRef = useRef(
+    <span className="text-vega-blue-500 dark:text-vega-blue-500">
+      <VegaIcon name={VegaIconNames.BULLET} />
+    </span>
+  );
+  const arrowIcon = useMemo(() => {
```

Comment:
Can you remove the useRef, useMemo and useCallback it here. Its unnecessary. Just compute it directly in the function

Have a read of this:  https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse